server/src/main/java/com/objectcomputing/checkins/services/reports/ReportHours.java

```diff
@@ -0,0 +1,14 @@
+package com.objectcomputing.checkins.services.reports;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+class ReportHours {
+  private final float contributionHours;
+  private final float ptoHours;
+  private final float overtimeHours;
+  private final float billableUtilization;
+}
+
```

Comment:
This can be a record

```suggestion
package com.objectcomputing.checkins.services.reports;

record ReportHours(
        float contributionHours,
        float ptoHours,
        float overtimeHours,
        float billableUtilization
) {
}
```